packages/plans-grid-next/src/components/features-grid/table.tsx

```diff
@@ -152,7 +152,7 @@ const Table = ( {
 				</tr>
 				{ enableCategorisedFeatures ? (
 					<>
-						<tr>
+						<tr className="plans-grid-next-features-grid__feature-group-row is-first-feature-group-row">
```

Comment:
I find this a little easier to grapple - meaning, "storage" being the first feature group row. Ultimately, down the line, we can consolidate storage and feature groups, and in the default (no categories) form we can have a `null` feature group along with "storage", which should will remove some of the complexity/conditioning.

Reply:
Adding a more semantically relevant class name looks great here :)